composer.json

```diff
@@ -40,7 +40,8 @@
         "psr/log": "~1.0",
         "monolog/monolog": "~1.6",
         "pimple/pimple": "~1.1",
-        "robmorgan/phinx": "~0.4"
+        "robmorgan/phinx": "~0.4",
+        "krausv/ftp-deployment": "v2.1-RC"
```

Comment:
You are referencing https://github.com/krausv/ftp-deployment that is a fork of the original project https://github.com/dg/ftp-deployment, why?